src/modules/head/head.service.ts

```
@@ -0,0 +1,27 @@
+import { Repository } from 'typeorm';
+import { HeadEntity } from 'src/database/entities/head.mongo-entity';
+import { CreateHeadDTO } from './dto/create-head-dto';
+
+@Injectable()
+export class HeadService{
+    constructor(
+        @InjectRepository(HeadEntity, 'mongoConnection')
+        private readonly headEntityRepo: Repository<HeadEntity>
+    ){}
+
+    async headIsRegistered(email: string): Promise<Boolean>{
```

Biome (1.9.4)

[error] Don't use 'Boolean' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'boolean' instead


(lint/complexity/noBannedTypes)